app/Services/Credit.php

```diff
@@ -90,6 +90,15 @@ public function getBalance()
         return $this->user->cash_balance / 100;
     }
 
+    /**
+     * Get the users balance sign (positive/negative)
+     * @return string
+     */
+    public function getBalanceSign()
+    {
+		return ( (int) $this->getBalance() >= 0 ? 'positive' : 'negative' );
```

Comment:
Not sure what happened there, I'm set with spaces, just seems to be the return line that was committed with tabs. Have updated the pull request.